Automattic/wp-calypso
# Help Center: What's New fix on mobile, help icon style

#### Changes proposed in this Pull Request

Fix What's New modal  on mobile.

| Before  | After |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52076348/167103873-d234d5fe-5f32-451b-ab08-19e1b0198f95.png) | ![image](https://user-images.githubusercontent.com/52076348/171136896-6d185a1a-e8d7-4237-b0f2-97f8614f76e8.png) |

Removed `active` from the help-icon, it is now managed by CSS. Fixed icon color on mobile.

#### Testing instructions



* Checkout branch
* `yarn dev --sync` from `apps/editing-toolkit`
* Click on `What's New` in the 3 dots menu
* Check the new design
* Make sure that the help icon is visible on mobile



Related to #63304
Fixes #63304
